docker/README.md

```diff
@@ -0,0 +1,90 @@
+# Docker images of HeAT
```

Comment:
As far as I understood, there are two options: either downloading and using a pre-build container (is there a link?) or to build a container yourself? And the following section "Docker images of HeAT" explains this second possibility... Is this correct?